### PR TITLE
fix: add ayushigeorge

### LINKED ISSUE #1504: Ayushi George
## Profile Screenshot

"https://user-images.githubusercontent.com/76419649/197496166-56b35008-600f-4638-8d15-44c788b687b0.jpg"

Issue id: #1504

### PR DESCRIPTION
**Pull request by ayushigeorge**
resolves #1504

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/ayushigeorge.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "![githubss](https://user-images.githubusercontent.com/76419649/197496166-56b35008-600f-4638-8d15-44c788b687b0.jpg)",
+  "issueId": 66,
```
Comment (Panquesito7) [inline profiles/ayushigeorge.json:3]:
```suggestion
  "image": "https://user-images.githubusercontent.com/76419649/197496166-56b35008-600f-4638-8d15-44c788b687b0.jpg",
  "issueId": 1504,
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by ayushigeorge** [comment]
Done the required changes.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**